bundle/edu.gemini.spModel.core/src/test/scala/edu/gemini/spModel/core/CoordinatesSpec.scala

```diff
@@ -55,6 +55,16 @@ object CoordinatesSpec extends Specification with ScalaCheck with Arbitraries wi
 
   }
 
+  "Coordinates Angular Offset" should {
+
+    "result in an angular separation equal to distance, regardless of bearing" !
+      forAll { (c: Coordinates, bearing: Angle, distance: Angle) =>
+        val newCoordinates = c.angularOffset(bearing, distance)
+        (c.angularDistance(newCoordinates) ~= distance) || (newCoordinates.angularDistance(c) + distance ~= Angle.zero)
```

Comment:
I would think `angularDistance` would always be positive.  Is that not the case?

Reply:
Hmmm... I thought I did that, but maybe it got lost in the git confusion. I'll look at these issues.